src/spikeinterface/core/recording_tools.py

```diff
@@ -910,3 +910,34 @@ def get_rec_attributes(recording):
         dtype=recording.get_dtype(),
     )
     return rec_attributes
+
+
+def check_recording_attributes_match(recording1, recording2_attributes, skip_properties=True) -> bool:
+    """
+    Check if two recordings have the same attributes
+
+    Parameters
+    ----------
+    recording1 : BaseRecording
+        The first recording object
+    recording2_attributes : dict
+        The recording attributes to test against
+
+    Returns
+    -------
+    bool
+        True if the recordings have the same attributes
+    """
+    recording1_attributes = get_rec_attributes(recording1)
+    recording2_attributes = deepcopy(recording2_attributes)
+    if skip_properties:
+        recording1_attributes.pop("properties")
+        recording2_attributes.pop("properties")
```

Comment:
Are upi handling the case skip_properties:=false after this ?
I would remove skip_properties option because we do not use.
No ?